var/spack/repos/builtin/packages/py-bglibpy/package.py

```diff
@@ -14,6 +14,9 @@ class PyBglibpy(PythonPackage):
 
     version('develop', branch='master')
     version('4.1.4', commit='e54d294460e5bdf6b9990bc10a4606b412b76d90')
+    version('4.1.2', commit='d01c9359edf92e39be5b840b5bb975da1c960988')
+    version('4.0.27', commit='42d9c1f891ef1ec9af6d72c49ff3b7726a009951')
+    version('4.0.17', commit='e90513f52a7d9ca3c16877ae7a2bcb8df31c8545')
```

Comment:
dont understand why these are here

Reply:
I think they were part of some rebase. I didn't touch them since I figured they would be picked out in this phase if they were useless. :smile: